.travis.yml

```diff
@@ -37,14 +37,17 @@ matrix:
       - PRODUCT_TESTS='-g hdfs_impersonation'
       - CLUSTER_TYPE='singlenode-hdfs-impersonation'
     - env:
-      - PRODUCT_TESTS='-g hdfs_impersonation,authorization'
+      - PRODUCT_TESTS='-g hdfs_impersonation,authorization,storage_formats,cli'
       - CLUSTER_TYPE='singlenode-kerberos-hdfs-impersonation'
     - env:
       - PRODUCT_TESTS='-g ldap_cli'
       - CLUSTER_TYPE='singlenode-ldap'
     - env:
       - MVN_TESTS='-Dtest=TestLocalBinarySpilledQueries,TestPostgreSqlDistributedQueries,TestMySqlDistributedQueries'
       - MVN_MODULES='-pl presto-tests,presto-mysql,presto-postgresql'
+    - env:
+      - PRODUCT_TESTS='-g hdfs_no_impersonation,storage_formats,cli'
```

Comment:
Let's hope for more travis resources then :)

As you've pointed out, this will only help for tests needing the same container - and I think the jobs using singlenode container would be on the verge of failing if we combined any two of them.